bindata/etcd/pod.yaml

```diff
@@ -244,6 +264,36 @@ ${COMPUTED_ENV_VARS}
         name: cert-dir
       - mountPath: /var/lib/etcd/
         name: data-dir
+  - name: health-check-probe
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: [ "cluster-etcd-operator", "health-probe" ]
+    args:
+      - --endpoint=https://localhost:2379
+      - --probe-interval=1s
+      - --log-outputs=stderr
+      - --log-outputs=/var/log/etcd/etcd-health-probe.log
+      - --enable-log-rotation
+      - --pod-name=${POD_NAME}
+      - --cert-file=${ETCDCTL_CERT}
+      - --key-file=${ETCDCTL_KEY}
+      - --cacert-file=${ETCDCTL_CACERT}
+    env:
+${COMPUTED_ENV_VARS}
+      - name: "ETCD_STATIC_POD_REV"
+        value: "REVISION"
+      - name: POD_NAME
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.name
+    volumeMounts:
+      - mountPath: /var/log/etcd/
+        name: log-dir
+    resources:
+      requests:
+        memory: 600Mi
```

Comment:
needs profiled and tuned.

Reply:
> needs profiled and tuned.

yeah... that should be a lot smaller :)

Reply:
still pending.

Reply:
![steady-state](https://user-images.githubusercontent.com/1249749/121400728-ea7d8680-c925-11eb-8242-07926c74575a.png)